src/clang.rs

```diff
@@ -448,7 +453,11 @@ impl Cursor {
 
     /// Get the visibility of this cursor's referent.
     pub fn visibility(&self) -> CXVisibilityKind {
-        unsafe { clang_getCursorVisibility(self.x) }
+        if clang_getCursorVisibility::is_loaded() {
+            unsafe { clang_getCursorVisibility(self.x) }
+        } else {
+            CXVisibility_Default
```

Comment:
Not sure if this will work for functions too? Have you checked `cargo test`? How does the diff look like?